spaces/S000115/properties/P000200.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000115
+property: P000200
+value: false
+---
+
+Consider the subspaces $X_1=X\cap [\frac{2}{\pi},1]\times\mathbb{R}$ and $X_2=X\cap [0,\frac{2}{\pi}]\times\mathbb{R}$.
+Now $X= X_1\cup X_2$, $X_1 \cap X_2 = \{(\frac{2}{\pi},1)\}$ and $X_i\backslash\{(\frac{2}{\pi},1)\}$ is open in $X$.
+Therefore we have $\pi_1(X) \cong \pi_1(X_1\wedge X_2) \cong \pi_1(X_1) \times \pi_1(X_2) \cong \pi_1(X_1) \times \pi_1(S^1) \cong \pi_1(X_1) \times \mathbb{Z}$ which is not trivial.
```

Comment:
```suggestion
Consider the subspaces $X_1=X\cap\left(\left[0,\frac{2}{3\pi}\right)\times\mathbb{R}\right)$ and $X_2=X\cap\left(\left(\frac{2}{7\pi},1\right]\times\mathbb{R}\right)$.
Now $X= X_1\cup X_2$, $X_1 \cap X_2 = X \cap \left( \left( \frac{2}{7\pi}, \frac{2}{3\pi} \right) \times \mathbb{R} \right)$ is simply connected and both $X_1, X_2$ are open.
Therefore we have $\pi_1(X) \cong \pi_1(X_1\vee X_2) \cong \pi_1(X_1) \times \pi_1(X_2) \cong \pi_1(X_1) \times \pi_1(S^1) \cong \pi_1(X_1) \times \mathbb{Z}$ which is not trivial.
```
* Wedge sum is denoted as `\vee` rather than `\wedge`.
* Van Kampen requires the open cover condition.
* Fix some typos.

Reply:
Most people here (including me) are not as familiar with algebraic topology as you are.  So it would help to mention "By Van Kampen's theorem" somewhere in the text.